### ISSUE TITLE
Twitter card image alt missing (Web Accessibility)

### ISSUE THREAD

**Issue by pacollins** [gohugoio/hugo#6260]
Currently the internal Twitter card template does not support [`twitter:image:alt`](https://blog.twitter.com/developer/en_us/a/2016/alt-text-support-for-twitter-cards-and-the-rest-api.html), which decreases web accessibility and could cause ADA compliance issues.

https://github.com/gohugoio/hugo/blob/c5319db9f13f1dee97db5fbbeae38429a074c7d0/tpl/tplimpl/embedded/templates/twitter_cards.html#L1-L29

My proposal is to take `images=[]` and `image=""` and do something to the effect of:
```
images = [["img1.jpg","img1 alt"],
         ["img2.jpg","img2 alt"], 
         ["img2.jpg","img3 alt"]]
```
